### PR TITLE
Fixes #15. Add support for separate hosts

### LINKED ISSUE #15: Support for "separate host"
According to [this page](https://cweiske.de/tagebuch/munin-multiple-hosts.htm), it is possible to let Munin think some parts of the data belongs to some sort of subhost, which is logical to have for these scripts. However, there is no option to introduce this. 

Currently, I've added the option to the scripts myself, but I've not yet been able to migrate my current history to the new "subhost'. However, it should work nicely to split the graphs.

I might create a PR for this in the coming week, depending on my time.

### PR DESCRIPTION
**Pull request by bobvandevijver**
By using the environment variable `host_name` you can now configure the configuration phase of the plugins, allowing to split the graphs in munin easily.

### CODE REVIEW

File: fritzbox_connection_uptime.py [hunk 1]
```diff
@@ -38,6 +38,8 @@ def print_config():
     print "graph_category network"
     print "uptime.label uptime"
     print "uptime.draw AREA"
+    if os.environ.get('host_name') != None:
```
Comment (Tafkas) [inline fritzbox_connection_uptime.py]: This can be simplified to

```
if os.environ.get('host_name'):
```

Comment (bobvandevijver) [reply fritzbox_connection_uptime.py]: You're right, adjustment on its way

**Comment by Tafkas** [comment]
Can you rebase this?

**Comment by bobvandevijver** [comment]
Done!

**Comment by Tafkas** [comment]
merged. Great stuff.

**Pull request merged**